Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -47,12 +48,38 @@ series to which the filter `ℱ` converges.
 `LaurentSeries.val_le_one_iff_eq_coe`.
 * The uniform space of `LaurentSeries` over a field is complete, formalized in the instance
 `instLaurentSeriesComplete`.
+************************************************************************************************************************
```

Comment:
Are you intentionally including 120 `*`s on this line?

Reply:
Yes, I was, to remind me I had to finish the doc... and then forgot to remove them 😭 Now fixed.